### PR TITLE
[2.0] update erfa imports for astropy

### PR DESCRIPTION
**Pull request by nabobalis**
We did not backport this.

### CODE REVIEW

**Review by kakirastern — APPROVED** [review]

File: tox.ini [hunk 1]
```diff
@@ -82,7 +82,6 @@ extras =
 commands =
     !online-!hypothesis-!figure: {env:PYTEST_COMMAND} {posargs}
     online: {env:PYTEST_COMMAND} --reruns 2 --timeout=180 --remote-data=any -m "remote_data" {posargs}
-    devdeps-!figure: bash -ec "for f in {toxinidir}/examples/*/*.py; do [[ $f == *skip* ]] && continue; echo "$f" && python "$f"; done"
```
Comment (nabobalis) [inline tox.ini:85]: Time to restore the devdeps as working now.

Comment (kakirastern) [reply tox.ini:85]: Cool 👍

**Comment by nabobalis** [comment]
Codecov failed on the 32 bit build.

**Comment by ayshih** [comment]
Er, what?  This is not a backport of the approach to importing `erfa` that was agreed upon in #4343.

**Pull request merged**